src/components/SearchPane/SideMenu/MenuComponents/EntityDetails/MobileDrawer/MobileDrawer.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useRef, useEffect } from 'react';
+import { IonCard, createGesture, IonIcon } from '@ionic/react';
+import '../entityDetails.css';
+import EntityDetails from '../EntityDetails';
+import { remove } from 'ionicons/icons';
+
+interface PageProps {
+    isOpen: boolean;
+    closeMenu: Function;
+    isSearching: boolean;
+    isMobile: boolean;
+    selectedLocation: any;
+}
+
+const MobileDrawer: React.FC<PageProps> = (props) => {
+    const drawerRef = useRef<any>();
+    const dragRef = useRef<any>();
+
+    const toggleDrawer = () => {
+        let c = drawerRef.current;
+        if (c.dataset.open === "true") {
+          c.style.transition = ".5s ease-out";
+          c.style.transform = "";
+          c.dataset.open = "false";
+        } else {
+          c.style.transition = ".5s ease-in";
+          c.style.transform = `translateY(${-68}vh) `;
+          // c.style.transform = `translateY(${-350}px) `;
+          c.dataset.open = "true";
+        }
+      };
+
+      const openDrawer = () => {
+        let c = drawerRef.current;
+
+        c.style.transition = ".5s ease-in";
```

Comment:
Do you think it would be better to do this with applying/removing a CSS class rather than direct in the code?